### ISSUE TITLE
[Accessibility] no clear focus outline for link on Windows, URL is provided as tooltip for the link, no information provided like 'opens in a new tab or window'

### ISSUE THREAD

**Issue by wanghe2015** [MicrosoftDocs/msteams-docs#9691]
### **Description**
When we use screen reader to test accessibility, we found:

- There's no clear focus outline for buttons on Windows, but there's clear focus outlines on MAC.

<img width="1591" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/13779512/2df5e958-f936-4940-93fa-4dbb01ea15d5">

- On accessing the links 'Pay Statement Details' and 'Pay History' an URL information is read out which is also provided in the tooltip.This URL announcement for these links seems to be additional announcements. 

- Also the tooltip do not have information such as "link opens in a new tab or window".
### **Expected Behavior**

- There should be clear focus outline for link both on Windows and MAC
- The additional url announcement for links can be avoided.
- The tooltip provided for links shall have information such as "link opens in a new tab or window".
### **Card Json**
{
  "task": {
    "type": "continue",
    "value": {
      "title": "[(${cardTitle})]",
      "width": "[(${cardWidth})]",
      "card": {
        "contentType": "application/vnd.microsoft.card.adaptive",
        "content": {
          "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
          "type": "AdaptiveCard",
          "version": "1.5",
          "body":[
            [# th:if="${emptyImageUrl} == ''"]
            {
              "id": "paystatement-payout-value",
              "type": "TextBlock",
              "text": [[${payOut}]],
              "wrap": true,
              "horizontalAlignment": "Center",
              "size": "Large",
              "style": "heading",
              "weight": "bolder"
            },
            {
              "id": "paystatement-payout-text",
              "type": "TextBlock",
              "text": "[(#{PAYROLL_PAYSTATEMENT_PAYOUT_TEXT})]",
              "wrap": true,
              "horizontalAlignment": "Center",
              "size": "Medium",
              "spacing": "Small",
              "isSubtle": false
            },
            {
              "type": "Table",
              "gridStyle": "default",
              "firstRowAsHeaders": false,
              "columns": [
                {
                "width": 1
                },
                {
                "width": 1
                }
              ],
              "rows": [
                {
                  "type": "TableRow",
                  "cells": [
                    {
                      "type": "TableCell",
                      "style": "default",
                      "items": [
                        {
                          "type": "TextBlock",
                          "wrap": true
                        }
                      ]
                    },
                    {
                      "type": "TableCell",
                      "style": "default",
                      "items": [
                        {
                          "type": "TextBlock",
                          "wrap": true
                        }
                      ]
                    }
                  ],
                  "style": "default"
                },
                {
                  "type": "TableRow",
                  "cells": [
                    {
                      "type": "TableCell",
                      "style": "default",
                      "items": [
                        {
                          "id": "paystatement-gross-text",
                          "type": "TextBlock",
                          "text": "[(#{PAYROLL_PAYSTATEMENT_GROSS_TEXT})]",
                          "wrap": true,
                          "fontType": "Default",
                          "weight": "Bolder",
                          "color": "Default",
                          "horizontalAlignment": "Left",
                          "size": "Medium"
                        }
                      ]
                    },
                    {
                      "type": "TableCell",
                      "style": "default",
                      "items": [
                        {
                          "id": "paystatement-gross-value",
                          "type": "TextBlock",
                          "text": [[${gross}]],
                          "wrap": true,
                          "horizontalAlignment": "Left",
                          "size": "Medium"
                        }
                      ]
                    }
                  ],
                  "style": "default"
                },
                {
                  "type": "TableRow",
                  "cells": [
                    {
                      "type": "TableCell",
                      "style": "default",
                      "items": [
                        {
                          "id": "paystatement-paydate-text",
                          "type": "TextBlock",
                          "text": "[(#{PAYROLL_PAYSTATEMENT_PAYDATE_TEXT})]",
                          "wrap": true,
                          "style": "default",
                          "weight": "Bolder",
                          "horizontalAlignment": "Left",
                          "size": "Medium"
                        }
                      ]
                    },
                    {
                      "type": "TableCell",
                      "style": "default",
                      "items": [
                        {
                          "id": "paystatement-paydate-value",
                          "type": "TextBlock",
                          "text": [[${payDate}]],
                          "wrap": true,
                          "horizontalAlignment": "Left",
                          "size": "Medium"
                        }
                      ]
                    }
                  ]
                }
              ],
              "showGridLines": false,
              "spacing": "ExtraLarge"
            },
            {
              "id": "paystatement-details-link-text",
              "type": "TextBlock",
              "text": "[(|[#{PAYROLL_PAYSTATEMENT_LINK_TEXT}](${link})|)]",
              "spacing": "ExtraLarge"
            },
        [/]
        [# th:if="${emptyImageUrl} != ''"]
        {
          "id": "paystatement-no-message-link",
          "type": "Image",
          "width": "160px",
          "height": "160px",
          "url": [[${emptyImageUrl}]],
          "horizontalAlignment": "Center",
          "separator": true,
          "spacing": "medium"
        },
        {
          "id": "paystatement-no-message-text",
          "type": "TextBlock",
          "text": "[(#{PAYROLL_PAYSTATEMENT_NO_MESSAGE_TEXT})]",
          "weight": "Bolder",
          "wrap": true,
          "size": "medium",
          "horizontalAlignment": "Center"
        },
        {
          "id": "paystatement-subtitle-no-message-text",
          "type": "TextBlock",
          "text": "[(#{PAYROLL_PAYSTATEMENT_SUBTITLE_NO_MESSAGE_TEXT})]",
          "wrap": true,
          "horizontalAlignment": "Center",
          "size": "medium",
          "spacing": "small",
          "isSubtle": true
        },
        {
          "id": "paystatement-space",
          "type": "TextBlock",
          "text": "\n\n\n\n\n\n\n",
          "spacing": "Large",
          "wrap": true
        },
      [/]
        {
          "id": "paystatement-history-deeplink-text",
          "type": "TextBlock",
          "text": "[(|[#{PAYROLL_PAYSTATEMENT_VIEW_PAYSTATEMENT_DEEPLINK_TEXT}](${deeplink})|)]"
        },
        {
          "type": "TextBlock",
          "text": "\n",
          "spacing": "Small",
          "wrap": true
        }
          ]
        }
      }
    }
  }
}

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this issue, we will check this and update you.
